api/sql/schema/750$transfer.transfer.get.sql

```diff
@@ -14,24 +17,41 @@ AS
             RAISERROR('transfer.missingParameter', 16, 1);
             RETURN 55555 
       END
+      
+    DECLARE @transfer AS TABLE (transferId bigint, transferTypeId bigint, acquirerCode varchar(50), transferIdAcquirer varchar(50), transferIdLedger varchar(50),
+                                transferIdIssuer varchar(50), transferIdMerchant varchar(50), transferDateTime datetime, localDateTime varchar(14), issuerSettlementDate date, channelId bigint,
+                                channelType varchar(50), ordererId bigint, merchantId varchar(50), merchantInvoice varchar(50), merchantPort varchar(50), merchantType varchar(50),
+                                cardId bigint, sourceAccount varchar(50), destinationAccount varchar(50), expireTime datetime, expireCount int, reversed bit, retryTime datetime,
+                                retryCount int, ledgerTxState smallint, issuerTxState smallint, acquirerTxState smallint, merchantTxState smallint, issuerId varchar(50), ledgerId varchar(50),
+                                transferCurrency varchar(3), transferAmount money,  acquirerFee money, issuerFee money, transferFee money,[description] varchar(250), issuerPort varchar(50),
+                                ledgerPort varchar(50),udfAcquirer xml)
+    
+    INSERT INTO @transfer
```

Comment:
A SELECT statement that assigns a value to a variable must not be combined with data-retrieval operations.